### PR TITLE
feat(docker): add docker deploy docs

### PR DESCRIPTION
**Pull request by zibuyu1995**

### CODE REVIEW

File: zh/installation/docker.md [hunk 1]
```diff
@@ -2,13 +2,147 @@
...
+
+```bash
+$ docker run -it -d --restart=always --name actorcloud \
+    -p 80:80 \
+    -p 7000:7000 \
+    -p 7001:7001 \
+    -v /opt/actorcloud/status:/opt/actorcloud/server/static \
+    -v /opt/actorcloud/instance:/opt/actorcloud/instance \
+    -v /opt/actorcloud/logs:/opt/tmp/logs \
+    -v /opt/nginx:/etc/nginx \
```
Comment (CrazyWisdom) [inline zh/installation/docker.md]: -v /opt/actorcloud/nginx:/etc/nginx \

File: zh/installation/docker.md [hunk 2]
```diff
@@ -2,13 +2,147 @@
...
+
+
+
+
+
+### 配置 ActorCloud
+
+#####数据卷说明
+
+* status/ : 静态文件存放目录(images, excels)
```
Comment (CrazyWisdom) [inline zh/installation/docker.md]: static?

File: zh/installation/docker.md [hunk 3]
```diff
@@ -2,13 +2,147 @@
...
+           /opt/actorcloud/instance \
+           /opt/actorcloud/logs \
+           /opt/actorcloud/nginx
+```
+
+
+
+### 容器部署
+
+##### 获取ActorCloud 镜像
```
Comment (oucb) [inline zh/installation/docker.md]: `获取ActorCloud 镜像` -> `获取 ActorCloud 镜像`

File: zh/installation/docker.md [hunk 4]
```diff
@@ -2,13 +2,147 @@
...
+
+```bash
+$ docker pull actorcloud:latest
+```
+
+
+
+#####运行容器
```
Comment (oucb) [inline zh/installation/docker.md]: `#####运行容器` -> `##### 运行容器`

Comment (Zhiw) [inline zh/installation/docker.md]: Add a space

File: zh/installation/docker.md [hunk 5]
```diff
@@ -2,13 +2,147 @@
...
+    actorcloud:latest
+```
+
+
+
+
+
+### 配置 ActorCloud
+
+#####数据卷说明
```
Comment (Zhiw) [inline zh/installation/docker.md]: Add a space

**Pull request closed without merging**